From aadd548c28e3bf4502fcbc4df0865296745183ed Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@petalogix.com>
Date: Thu, 26 Jan 2012 10:21:45 +1000
Subject: [PATCH 12/41] microblaze: stackinfo.h Define DEFAULT_STACK_PERMS

prevents;
dl-load.c:1084:32: error: 'DEFAULT_STACK_PERMS' undeclared (first use in this function)

Signed-off-by: David Holsgrove <david.holsgrove@petalogix.com>
---
 sysdeps/microblaze/stackinfo.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sysdeps/microblaze/stackinfo.h b/sysdeps/microblaze/stackinfo.h
index 819e819..f4a8dcd 100644
--- a/sysdeps/microblaze/stackinfo.h
+++ b/sysdeps/microblaze/stackinfo.h
@@ -24,5 +24,7 @@
 
 /* On microblaze the stack grows down.  */
 #define _STACK_GROWS_DOWN	1
+/* PF_X can be overridden if PT_GNU_STACK is present but is presumed absent.  */
+# define DEFAULT_STACK_PERMS (PF_R|PF_W|PF_X)
 
 #endif	/* stackinfo.h */
-- 
1.7.1

